api/scpca_portal/management/commands/configure_aws_cli.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import subprocess
+
+from django.core.management.base import BaseCommand
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.StreamHandler())
+
+
+class Command(BaseCommand):
+    help = """Configures the AWS CLI.
+
+    Current supported options include:
+        --s3-max-bandwidth
+        --s3-max-concurrent-requests
+        --s3-multipart-chunk-size
+
+    The configuration works by writing the config values to ~/.aws/config,
+    which is references by the AWS CLI tool each time its called."""
+
+    def add_arguments(self, parser):
+        parser.add_argument("--s3-max-bandwidth", type=int, default=None, help="In MB/s")
+        parser.add_argument("--s3-max-concurrent-requests", type=int, default=10)
+        parser.add_argument("--s3-multipart-chunk-size", type=int, default=8, help="In MB")
+
+    def handle(self, *args, **kwargs):
+        self.configure_aws_cli(**kwargs)
+
+    def configure_aws_cli(self, **kwargs):
```

Comment:
lets define the keyword arguments here instead of **kwargs

Reply:
Unfortunately Python doesn't have an analogous feature to ES6's destructuring assignment, where you can do the following:
```python
def handle(self, *args, **kwargs):
    s3_max_bandwidth, s3_max_concurrent_requests, s3_multipart_chunk_size, ...rest = kwargs
    self.configure_aws_cli(s3_max_bandwidth, s3_max_concurrent_requests, s3_multipart_chunk_size)

def configure_aws_cli(
    self,
    *,
    s3_max_concurrent_requests: int = 10,
    s3_multipart_chunk_size: int = 8,
    s3_max_bandwidth: int,
) -> None:
    ...
```

The closest thing that I found was the following (from a stack overflow [link](https://stackoverflow.com/questions/2955412/destructuring-bind-dictionary-contents)):
```python
from operator import itemgetter

params = {'a': 1, 'b': 2}

a, b = itemgetter('a', 'b')(params)
```

For now I went with the following, but I'd be interesting to hear your feedback:
```python
def handle(self, *args, **kwargs):
    self.configure_aws_cli(**kwargs)

def configure_aws_cli(
    self,
    *,
    s3_max_concurrent_requests: int = 10,
    s3_multipart_chunk_size: int = 8,
    s3_max_bandwidth: int,
    **kwargs,
):
    ...
```

Reply:
yeah what you went with is correct, but you dont need to pass **kwargs since we dont handle them